pkg/util/annotation.go

```diff
@@ -21,3 +21,11 @@ func MergeAnnotations(dst *unstructured.Unstructured, src *unstructured.Unstruct
 		MergeAnnotation(dst, key, value)
 	}
 }
+
+// GetAnnotationValue retrieves the value via 'annotationKey' if exist, otherwise returns an empty string.
+func GetAnnotationValue(annotations map[string]string, annotationKey string) string {
```

Comment:
Why is this needed? if key doesn't exist, `map[key]` will be "" naturally

Reply:
The annotations may be nil.